integration/e2e/util.go

```diff
@@ -86,12 +86,21 @@ func PostRequest(url string) (*http.Response, error) {
 	return client.Post(url, "", strings.NewReader(""))
 }
 
-// timeToMilliseconds returns the input time as milliseconds, using the same
+// TimeToMilliseconds returns the input time as milliseconds, using the same
```

Comment:
**suggestion(optional):** To give context to future readers it may be worth adding a comment with a permalink to the [Prometheus parseTime function](https://github.com/prometheus/prometheus/blob/df80dc4d3970121f2f76cba79050983ffb3cdbb0/web/api/v1/api.go#L1690-L1694) this is referencing. There is a `FromTime` function in the Prometheus timestamp package that looks like it would accomplish the same thing but doesn't round the same way. However, I imagine someone reading may be unclear if the method below is up to date as Prometheus changes.

Reply:
Good point, done.